index.html

```diff
@@ -27,7 +27,7 @@
     <div class="container">
       <div class="header">
         <ul class="nav nav-pills pull-right">
-          <li><a href="http://pyconse.tumblr.com">Blog</a></li>
+          <li><a href="#cfp">Call For Proposals</a></li>
           <li><a href="sponsorship.html">Interested in sponsoring?</a></li>
```

Comment:
Maybe we should remove the sponsorship link until I've fixed some levels for this year